### PR TITLE
Install Dokku on a clean server

### LINKED ISSUE #7: [Feature] Install Dokku on a clean server

### PR DESCRIPTION
**Pull request by jvanbaarsen**
This removes the requirement that the end user has to install Dokku themselves.
We do it for them! This also gives us the oppertunity to install plugins that we
think are useful (dokku-intercity in the future?)

Fixes #7 

/cc @brambokdam

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -29,6 +29,7 @@ Vagrant::configure("2") do |config|
   end
 
   config.vm.define "clean" do |vm|
```
Comment (brambokdam) [inline Vagrantfile]: Is the `dokku` virtual machine unused now? If this is the case maybe it's a idea to remove it. Also maybe its useful to make the `clean` virtual machine the primary one.

Comment (jvanbaarsen) [inline Vagrantfile]: Hm I think it would be useful to keep it around, but make the clean the default would be good idea!

**Comment by brambokdam** [comment]
Looks great :shipit:!

**Comment by jvanbaarsen** [comment]
Thanks for the review :)

**Pull request merged**